.gitignore

```diff
@@ -13,3 +13,6 @@ packages/
 
 #ignore Resharper folders
 _ReSharper*/
+/.vs/Open.Nat/DesignTimeBuild
```

Comment:
Could you revert this change, please? Otherwise, probably could be a good idea to use a pattern like `.vs`